File: solid-principles/java/java-lsp-car/Ferrari.java
public class Ferrari extends Car implements IKlima, ITurbo{
    public Ferrari(int var1, int var2){
        super(var1, var2);
    }

    public void power(){
        this.acceleration = 10;
    }
 
    public void boost(){
        this.acceleration *= 2;
    }

    public void open(){
        this.acceleration *= 0.9;
    }
}

File: solid-principles/python/python-lsp-car/Ferrari.py
from Car import Car
from AirConditioning import AirConditioning
from Boost import Boost

class Ferrari(Car,AirConditioning,Boost):
    def power(self):
        self.acceleration=10
    def turbo(self):
         self.acceleration*=2
    def open(self):
        self.acceleration*=0.9
             